source/_docs/settings-php.md

```diff
@@ -112,27 +112,29 @@ Yes, but only if at least one other file (e.g. `settings.php`) is present within
 
 #### How can I write logic based on the Pantheon server environment?
 
-Depending on your use case, there are two possibilities:
+Depending on your use case, there are three possibilities:
 
-For web only actions, like redirects, check for the existence of $\_SERVER['PANTHEON\_ENVIRONMENT']. If it exists, it will contain a string with the current environment (Dev, Test, or Live).
+ - For web only actions, like redirects, check for the existence of `$\_SERVER['PANTHEON\_ENVIRONMENT']`. If it exists, it will contain a string with the current environment (Dev, Test, or Live):
 
-    // Pantheon - web only.
-    if (isset($_SERVER['PANTHEON_ENVIRONMENT'])) {
-      // Only on dev web environment.
-      if ($_SERVER['PANTHEON_ENVIRONMENT'] == 'dev') {
-        // Custom code.
-      }
-    }
+        // Pantheon - web only.
+        if (isset($_SERVER['PANTHEON_ENVIRONMENT'])) {
+          // Only on dev web environment.
+          if ($_SERVER['PANTHEON_ENVIRONMENT'] == 'dev') {
+            // Custom code.
+          }
+        }
 
-For actions that should take place on every environment, such as Redis caching, use the constant ​PANTHEON\_ENVIRONMENT. Again, it will contain Dev, Test, or Live.
+ - For actions that should take place on every environment, such as Redis caching, use the constant `PANTHEON\_ENVIRONMENT`. Again, it will contain Dev, Test, or Live:
 
-    // Pantheon - all operations.
-    if (defined('PANTHEON_ENVIRONMENT')) {
-      // Only on dev environment.
-      if (PANTHEON_ENVIRONMENT == 'dev') {
-        // Custom code.
-      }
-    }
+        // Pantheon - all operations.
+        if (defined('PANTHEON_ENVIRONMENT')) {
+          // Only on dev environment.
+          if (PANTHEON_ENVIRONMENT == 'dev') {
+            // Custom code.
+          }
+        }
+
+ - For Actions that require access to protected services like Redis or the site database, you can use the `$_ENV` superglobal. Please review our guide on [Reading Pantheon Environment Configuration](/docs/read-environment-config/) for more information.

```

Comment:
Consider moving this to the second option, since it's more recommended than $_SERVER

Suggest adding example code snippet to demonstrate usage, such as: 

```
// Handle Hash Salt Value from Drupal 8
if (isset($_ENV['PANTHEON_ENVIRONMENT'])) {
  $settings['hash_salt'] = $_ENV['DRUPAL_HASH_SALT'];
}
```